src/App.tsx

```diff
@@ -21,14 +13,41 @@ const styles = StyleSheet.create({
   },
 });
 
-interface IProps {}
+interface IProps {
+  decreaseCount: (counterId: number) => void,
+  increaseCount: (counterId: number) => void,
+  counters: ICounter[]
+}
 interface IState {}
```

Comment:
If a component doesn't have any state or lifecycle methods then it should be made into a "functional" component. (Actually with hooks now available we should be almost able to get away with only "functional" components. But the current version of Live is on RN 52 which uses a version of React that doesn't support hooks. 😢 ) e.g.
```
const App = (props: IProps) => {
  return ...
};
```
It removes a bunch of the boilerplate of class components.